src/ui/utils/logrocket.ts

```diff
@@ -4,22 +4,21 @@ import * as Sentry from "@sentry/react";
 import { skipTelemetry } from "./environment";
 import { Recording } from "ui/types";
 import { UserInfo } from "ui/hooks/users";
-import { AuthContext } from "ui/utils/useAuth0";
 
 let setup = false;
 
 export default {
   createSession: ({
     recording,
     userInfo,
-    auth,
+    auth0User,
   }: {
     recording?: Recording;
-    userInfo: UserInfo;
-    auth: AuthContext;
+    userInfo?: Omit<UserInfo, "loading">;
+    auth0User: any;
```

Comment:
This could use a type but I see that auth0 doesn't type it by default which is kinda lame.